scripts/output/vue.js

```diff
@@ -2,17 +2,31 @@ import { mkdirSync, writeFileSync } from 'node:fs'
 import { join as joinPath } from 'node:path'
 import { getSVGs, getDirname } from '../util/helpers.js'
 import chalk from 'chalk'
+import defaultIconDescriptions from '../../default-icon-descriptions.js'
 
 const __dirname = getDirname(import.meta.url)
 const icons = []
 const basepath = joinPath(__dirname, '../../vue/')
 mkdirSync(basepath, { recursive: true })
 
-getSVGs().forEach(({ svg, filename, exportName }) => {
+// Create Vue Icon
+getSVGs().forEach(({ svg, filename, exportName, name }) => {
+    // Handle i18n of icon title
```

Comment:
This comment for me doesn't add any useful information as the next line is actually to create an icon name and not handle i18n. So maybe it is better to remove this one or move it somewhere else, before line 20 for example. Wdyt?